pom.xml

```diff
@@ -36,6 +32,9 @@
                                     <requireJavaVersion>
                                         <version>${java.version}</version>
                                     </requireJavaVersion>
+                                    <requireMavenVersion>
+                                        <version>[3.0.5,)</version>
```

Comment:
is this version correct?

Reply:
These settings were copied from our DSpace/DSpace POM: https://github.com/DSpace/DSpace/blob/master/pom.xml#L71-L73

So, yes, they should work fine. Most people should have a much newer version of Maven than 3.0.5 though at this point.